src/validation_utils.py

```diff
@@ -55,3 +97,46 @@ def check_column_clusters(cols: dict, data_frame: pd.DataFrame):
                 if not pd.isna(row).all():
                     result.append((index, items))
     return result
+
+def unique_values(df: pd.DataFrame, col_name: str):
```

Comment:
I suppose unique implies no missing values? This should be somewhere in the documentation if this assumption is made. If not, then this logic is incorrect I think.